draw.io-api/pom.xml

```diff
@@ -139,4 +254,16 @@
       </plugin>
     </plugins>
   </build>
+  <repositories>
+    <repository>
+      <id>jgraph-drawio</id>
+      <url>https://packagecloud.io/jgraph/drawio/maven2</url>
```

Comment:
Would be good to add a comment explaining why it's needed.

Reply:
This remained from when I had `com.mxgraph:gae-stub` as dependency. Deleting it